src/read.c

```diff
@@ -2788,10 +2759,6 @@ void ReadEvalError(void)
 
 
 struct SavedReaderState {
-  Obj                 stackNams;
-  UInt                readTop;
-  UInt                readTilde;
-  UInt                currLHSGVar;
   UInt                userHasQuit;
   syJmp_buf           readJmpError;
   UInt                intrCoding;
```

Comment:
Hmm, I guess we should rename `SavedReaderState` to something else... 

Note that I am thinking about moving `STATE(IntrCoding)` and friends into a new `InterpreterState` (which ultimately also would be stack allocated...)

Reply:
This was done in PR #3823 which also removed `SavedReaderState`